### DISCUSSION TITLE
Multiple channels (of the same user)

### DISCUSSION THREAD

**Discussion by saschafoerster** [owncast/owncast#3546] [Ideas] [answered]
I understood that every OwnCast channel will be owned by one user and that no multi user configuration in Owncast is planned. 
Would it still be possible to have multiple "channels", so topics of different streams can be separated? So they get a folder-link like https://example.com/channel1, https://example.com/channel2, etc. 
When streaming to channel2, I could follow it as @channel2@example.com and the message sent is individually set up for this channel. Background: we are one company, but have different events and brands. Of course, a workaround would be to use simply multiple instances of Owncast, right?

**Comment by gabek** [comment, marked as answer]
Yeah, currently, people just run multiple instances of Owncast. And that way you can give each its own unique hostname as well.

While I know _technically_ it's not multiple _users_ that you're talking about, it pretty much is. It's just multiple users owned by the same person. Owncast just isn't built for that, and would require almost everything to be changed to support something like that.

One thing that may be a compromise, however, is with scheduled streams the hope is to have a unique page for every scheduled event with unique metadata. And while you couldn't follow a different user on the Fediverse, this unique metadata for this event then would be used for reminders and other page information.  So if you had event A every Monday at 6:00 it would have one page you could send people to, and event B every Saturday at 8:00 could have another page. But it would still all be owned by the same person. But unfortunately this work hasn't been started yet.
